src/components/SignupFinish.js

```diff
@@ -42,6 +66,7 @@ function SignupFinish({ backend }) {
             return;
           }
 
+          // service todo: is this the correct route for logging-in uer that doesn't have any service requests?
```

Comment:
a logged in user with no requests should go to `SERVICE_TYPE` to start the services flow right?

Reply:
Cool, that sounds good.